datasets/iowa_liquor_sales/docs/tutorials/liquor_sales_prediction/tutorial.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    gcp_project = 'gcp-public-data-contributors'
Remove "gcp-public-data-contributors" from this line and replace it with "YOUR-GCP-PROJECT" instead.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/public-datasets-pipelines/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>